docs/how-to-guides/fixing-dependent-package-versions.md

```diff
@@ -0,0 +1,10 @@
+# Fixing dependent package versions
```

Comment:
Please also update `.pages` or the index page.

Reply:
Resolved in [7d27537](https://github.com/autowarefoundation/autoware-documentation/pull/295/commits/7d27537fa40a1f52d53fa99c3bb101b57a214b2b)